content/news/050/index.md

```diff
@@ -76,56 +101,39 @@ If needed, a section can be split into subsections with a "------" delimiter.
 [Way of Rhea][wor] just got a release date: it will be coming to Steam on
 **May 20th, 2024**!
 
-Way of Rhea is a color based puzzle game with difficult puzzles, but forgiving
-mechanics being developed by [@masonremaley] in a custom Rust engine.
+Way of Rhea ([Steam](https://store.steampowered.com/app/1110620/Way_of_Rhea/)) is a color based puzzle game with difficult puzzles, but forgiving
+mechanics being developed by [@masonremaley] in a custom engine.
 
 You can support development by [wishlisting Way of Rhea on Steam][wor], or
 [signing up for the mailing list][wor-mail].
 
-Recent updates:
+The most important updates are:
 - A [closed beta][wor-closed-beta] began
 - All characters now have voices
-- A [reset button][wor-reset] was added for when undo is inconvenient
-- Speech bubbles now [identify the speaker][wor-speech-bubble]
-- Orbs now are shown with an "xray" effect so they are visible even when behind scenery
-- Older CPUs without AVX support are [now supported][wor-avx]
 - Various [speedrunning features][wor-speedrun] were added
-- Secrets were made harder
-- An [in game achievement UI][wor-achievements] was created
-- An [in game Jukebox UI][wor-jukebox] was created
-- For more updates, see the [release notes on Steam][wor-release-notes]...
+
+For the full changelog, see the [release notes][wor-release-notes].
```

Comment:
Instead of removing this list, can we turn it into a paragraph if it comes out too long? It isn't great removing someone's content after it was merged. Will make a suggestion comment.

Reply:
Actually there isn't a great way for me to suggest over this whole block, so I'll just post it here:
```
Recently, a [closed beta][wor-closed-beta] began. All characters now have
voices, and various [speedrunning features][wor-speedrun] were added. A [reset
button][wor-reset] was added for when undo is inconvenient. Speech bubbles now
[identify the speaker][wor-speech-bubble], and orbs are now shown with an "xray"
effect so they are visible even when behind scenery. Older CPUs without AVX
support are [now supported][wor-avx]. Secrets were made harder, and an [in-game
achievement UI][wor-achievements] was created. An [in-game Jukebox
UI][wor-jukebox] was also created.

For the full changelog, see the [release notes][wor-release-notes].
```

Reply:
I intentionally removed some content after feedback that the newsletter contained too many details most people don't care about. Personally, as a contributor, I would expect the editors to chop up my content however they think is best before release.

Reply:
re: community feedback, is that part of what's coming in from the survey, or was it discussed somewhere else?

I think this is fair though, the new paragraph looks good. It would be good for us to do a pass over instructions of writing sections for the newsletter so that any feedback is incorporated, once that all comes out 💯